erpnext/accounts/doctype/payment_reconciliation/payment_reconciliation.py

```diff
@@ -180,8 +232,6 @@ def check_condition(self):
 
 def reconcile_dr_cr_note(dr_cr_notes, company):
 	for d in dr_cr_notes:
-		voucher_type = ('Credit Note'
-			if d.voucher_type == 'Sales Invoice' else 'Debit Note')
 
```

Comment:
Can you confirm if we want the voucher_type in credit/debit_note reconciliation JE to be "Credit/Debit Note".
If yes, then revert these changes and add the "Credit Note" and "Debit Note" options (preferably with "Reconciliation") in voucher_type field in JE.
Else, it looks good.